src/aliceVision/image/caching.cpp

```diff
@@ -100,8 +102,10 @@ void ImageCache::add(const CacheKey& key, std::shared_ptr<Image<RGBAfColor>> img
     _memUsage.contentSize += img->MemorySize();
 }
 
-std::string ImageCache::toString() const
+std::string ImageCache::toString()
```

Comment:
`_mutex` should be mutable because it doesn't change the state of the object.

Reply:
Thanks that's what I was looking for !